authentication/src/main/AndroidManifest.xml

```diff
@@ -47,7 +47,7 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="com.uber.sdk.android.rides.samples.uberauth"
+                <data android:scheme="${applicationId}"
```

Comment:
This would be the applicationID from the consumer app including it? Any other hardcoded locations?

Reply:
correct. No, this is the only one that got through in my testing.